src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5439,10 +5439,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Argument cannot be used as an output for parameter due to differences in the nullability of reference types.</value>
   </data>
   <data name="WRN_DisallowNullAttributeForbidsMaybeNullAssignment" xml:space="preserve">
-    <value>A possible null value may not be assigned to a target marked with the [DisallowNull] attribute</value>
+    <value>A possible null value may not be assigned to a target marked with the [DisallowNull] attribute or output to a target marked with the [NotNull] attribute</value>
```

Comment:
>output to a target marked with [](start = 109, length = 30)

Is this qualifier necessary? (Is it necessary to differentiate "assigned to" and "output to"? Also, "output to" feels like "yield".)